bb-imager-gui/src/helpers.rs

```diff
@@ -376,6 +376,33 @@ pub(crate) fn system_timezone() -> Option<&'static String> {
     (*SYSTEM_TIMEZONE).as_ref()
 }
 
+pub(crate) fn system_keymap() -> String {
+    static SYSTEM_KEYMAP: LazyLock<Option<String>> = LazyLock::new(|| {
+        let lang = whoami::langs().ok()?.next()?;
+        let lang_str = lang.to_string();
+
+        let base = lang_str.split('.').next().unwrap_or(&lang_str);
+        let mut parts = base.split(|c| c == '-' || c == '_' || c == '/');
+
+    let _lang_part = parts.next().unwrap_or("us");
+        if let Some(region) = parts.next() {
+            let region = region.split('@').next().unwrap_or(region).trim();
+            let region_lower = region.to_lowercase();
+            if !region_lower.is_empty() {
+                if let Some(&canon) = crate::constants::KEYMAP_LAYOUTS
+                    .iter()
+                    .find(|k| k.eq_ignore_ascii_case(&region_lower))
+                {
+                    return Some(canon.to_string());
+                }
+            }
+        }
+
+        Some(String::from("us"))
+    });
+    (*SYSTEM_KEYMAP).as_ref().cloned().unwrap()
```

Comment:
No, this should not unwrap. Either return a string directly from `SYSTEM_KEYMAP` or use `unwrap_or`.